### DISCUSSION TITLE
How to prefill fields on create with ownerRecord data?

### DISCUSSION THREAD

**Discussion by Dontorpedo** [filamentphp/filament#4855] [Help] [answered]
Hey,

On the ownerRecord i create a new record with the RelationshipManager and would like to prefill some fields like Richeditor, TextInput and so on.. with already existing data from ownerRecord?

thanks

**Comment by Dontorpedo** [comment, marked as answer]
forgot to mention, as my form is pretty complex i dont use the modal form on the ownerRecord.. i redirct like so:

`                CreateAction::make()
                    ->action(null)
                    ->url(fn ($livewire) => ObjectResource::getUrl('create', ['ownerRecord' => $livewire->ownerRecord->getKey()]))
            ])`

so on the creationpage i can't use $livewire->ownerRecord bc it says: Property [$ownerRecord] not found

**Reply by danharrin** [reply]
So something like `TextInput::make('field_name')->default(OwnerModel::find(request()->query('ownerRecord'))?->field_name)` ?

**Reply by Dontorpedo** [reply]
yeah, thank you so much :D
